src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -78,3 +79,36 @@ export const getTeamReputationChartData = (
 
   return topTeams;
 };
+
+export const getContributorReputationChartData = (
+  contributorsList: ContributorItem[],
+): ReputationChartDataItem[] => {
+  const topContributors = contributorsList
+    .slice(0, WIDGET_TEAM_LIMIT)
+    .map(({ walletAddress, user, reputation }, index) => {
+      return {
+        id: walletAddress,
+        label: user?.profile?.displayName || '',
+        value: Number(reputation),
```

Comment:
By applying the above-mentioned filter, we'll be sure all contributors have reputation and we don't get `NaN` here

Reply:
Great point! Thank you! 
Unfortunately, I can't do that because we have types from graphQL so it was done for that purpose :( 
<img width="471" alt="image" src="https://github.com/user-attachments/assets/722dd5f6-7523-4fc8-a570-654cfcd177d5">